src/runtime/rpc/rpc_module.cc

```diff
@@ -177,6 +177,10 @@ class RPCModuleNode final : public ModuleNode {
   const char* type_key() const final { return "rpc"; }
 
   PackedFunc GetFunction(const std::string& name, const ObjectPtr<Object>& sptr_to_self) final {
+    if (name == "Shutdown") {
```

Comment:
As I'm reading it, I think we should rename the exposed function to something more descriptive, such as `CloseRPCConnection`.  I know we initially named it `Shutdown` because that's what the `RPCEndpoint` function is called, but it could cause confusion at this level.  (e.g. "Shutdown" could also mean to power off the remote device.)